CHANGELOG.md

```diff
@@ -62,9 +62,10 @@
   `collections.abc.Mapping`. No support for instances of `dict` that implement
   `__missing__` but are not instances of `collections.defaultdict`.
 - Added support for `DataFrame.align` and `Series.align` for `axis=1` and `axis=None`.
-- Added support fot `pd.json_normalize`.
+- Added support for `pd.json_normalize`.
 - Added support for `GroupBy.pct_change` with `axis=0`, `freq=None`, and `limit=None`.
 - Added support for `DataFrameGroupBy.__iter__` and `SeriesGroupBy.__iter__`.
+- Added support for `np.sqrt`, `np.trunc`, `np.floor`, numpy trig functions, `np.exp`, `np.abs`, `np.positive` and `np.negative`.
```

Comment:
nit: I think specifying the trig functions like sin, cos, sinh, etc is better than saying "trig functions" here -- easier to track for future reference.